Loop/Managers/NightscoutDataManager.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NightscoutDataManager.swift
+//  Loop
+//
+//  Created by Nate Racklyeft on 8/8/16.
+//  Copyright © 2016 Nathan Racklyeft. All rights reserved.
+//
+
+import Foundation
+import NightscoutUploadKit
+
+
+class NightscoutDataManager {
+
+    unowned let deviceDataManager: DeviceDataManager
+
+    init(deviceDataManager: DeviceDataManager) {
+        self.deviceDataManager = deviceDataManager
+
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(uploadLoopStatus(_:)), name: LoopDataManager.LoopDataUpdatedNotification, object: deviceDataManager.loopManager)
+    }
+
+    @objc func uploadLoopStatus(note: NSNotification) {
+
```

Comment:
Yeah, I'm going to need things like recommendedTempBasal, lastLoopError, eventualBG, iob, lastTempBasal, and glucoseVal. 

There's a need to have those items reflect the values actually used when making the recommendation; I may have cut corners with things like asking glucoseStore for latestGlucose, but what I really want is what was the latestGlucose when the forecast and recommendations were made. 

Would private vars on LoopDataManager be the place to accrete those (some are already there), and then would the userInfo be the mechanism by which to pass them to NightscoutDataManager?  Seems like a lot of fields to do that way.  Maybe some struct to handle all of that?